core/services/mongodb-remote/src/test/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizerUnitTests.kt

```diff
@@ -369,7 +369,7 @@ class DataSynchronizerUnitTests {
             expectedChangeEvent = ChangeEvent.changeEventForLocalUpdate(
                 ctx.namespace,
                 ctx.testDocumentId,
-                ctx.updateDocument,
+                ChangeEvent.UpdateDescription(BsonDocument("count", BsonInt32(2)), listOf()),
```

Comment:
[nit] you reuse this exact update in a lot of places so maybe just set a `testUpdateDescription` constant?

Reply:
Could, though for the purpose of test code, I think it's better to write in line. Though it's reused, it's fairly arbitrary, and throughout the test, the count can increase.